department-of-veterans-affairs/va.gov-cms
# VACMS: 10633 Translation permissions demo

## Description

Closes #10633.

Scope: Workflow and permission considerations around translation, not the FE or nuances of translation itself. (For example, Tagalog isn't in the Drupal community yet, etc.)

## Testing done
- [x] Manual functional testing

## Screenshots

## QA steps
- What needs to be checked to prove this works?  See below (as user...)
- [ ] What needs to be checked to prove it didn't break any related things? TODO (Exponential increase in content, FE concerns.)
- What variations of circumstances (users, actions, values) need to be checked?  Ensure that translator role can't edit existing content (see below)

As user _uid_ with user role "Content Translator - Spanish"
1. Go to Content > Resources & Support Type and edit any item.
2. See that there is a "Translate" tab.
3. Ensure that a new version for the Spanish translation shows (It will look the same/in English)
4. Run the Title and a paragraph field (or all fields) through Google translate (for the purpose of the test).
5. Click Save

The new translation should save without errors.

6. See if you can now edit the same content.

Note: You will need to give yourself the "Content translation - Spanish" role or use the dummy user [TODO dummy user with role] in demo.

#### Review documentation
Details are in the ticket in this comment:
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10633#issuecomment-1256795048

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
